src/components/periodic-report/periodic-report.resolver.ts

```diff
@@ -53,11 +53,18 @@ export class PeriodicReportResolver {
     return await this.service.update(input, session);
   }
 
-  @ResolveField(() => SecuredFile)
-  async reportFile(
+  @ResolveField(() => SecuredDirectory)
+  async reportDirectory(
     @Parent() report: IPeriodicReport,
     @AnonSession() session: Session
-  ): Promise<SecuredFile> {
-    return await this.files.resolveDefinedFile(report.reportFile, session);
+  ): Promise<SecuredDirectory> {
+    return {
+      canEdit: false,
+      canRead: true,
+      value: await this.files.getDirectory(
+        report.reportDirectory.value!,
+        session
+      ),
+    };
```

Comment:
We should forward on the auth properties just like before.
https://github.com/SeedCompany/cord-api-v3/blob/master/src/components/file/file.service.ts#L481-L490

It's a bit ambiguous but `canEdit` could be interrupted as can files be uploaded.